simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/Station.java

```diff
@@ -140,7 +140,7 @@ public void sendMessage(StationMessage stationMessage) {
      * @return {@link StationState}
      */
     public StationState getStateView() {
-        return state.getView();
+        return state;
```

Comment:
You've decided to expose inner state as is. Do you want to make it possible to change from the outside?

Reply:
We can make a copy of it if we need that. You can change the state from other thread as the StationState class is thread-safe. Shall we make a copy?